### PR TITLE
Add Migratefile JS variant documentation.

### PR DESCRIPTION
**Pull request by serhalp**
Follow-up to https://github.com/goodeggs/node-migrate-mongo/pull/13.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,13 @@ migrate generate --name my_first_migration
...
 ```
 
-## Migratefile Options
+## Migratefile
+
+### JavaScript variants
+
+Your Migratefile file can be any JavaScript variant recognized by [interpret](https://www.npmjs.com/package/interpret#jsvariants). Just make sure to use the right file extension. For CoffeeScript, use the `.coffee` extension. For babel.js, use the `.babel.js` extension.
+
+### Options
```
Comment (murkey) [inline README.md]: Are these command line options or functions that go in the file?

File: README.md [hunk 2]
```diff
@@ -26,7 +26,13 @@ migrate generate --name my_first_migration
 migrate all
 ```
 
-## Migratefile Options
+## Migratefile
+
+### JavaScript variants
+
+Your Migratefile file can be any JavaScript variant recognized by [interpret](https://www.npmjs.com/package/interpret#jsvariants). Just make sure to use the right file extension. For CoffeeScript, use the `.coffee` extension. For babel.js, use the `.babel.js` extension.
```
Comment (murkey) [inline README.md]: Super nitpicky but I think Babel is usually just referred to as Babel.

File: README.md (hunk 1 above)
Comment (serhalp) [reply README.md]: This is the "Options" subsection under the "Migratefile" section. Seems pretty self-explanatory that they're options that go in the Migratefile, no?

File: README.md (hunk 2 above)
Comment (serhalp) [reply README.md]: Sure, updated.

**Pull request merged**